cmd/root.go

```diff
@@ -116,6 +116,10 @@ func getConfigDir() string {
 }
 
 func initConfig() {
+	cfgFileFromEnv := os.Getenv("FIOCTL_CONFIG")
+	if len(cfgFileFromEnv) > 0 {
+		cfgFile = cfgFileFromEnv
+	}
 	if cfgFile != "" {
 		// Use config file from the flag.
 		viper.SetConfigFile(cfgFile)
```

Comment:
I'm fine with it.

The only question is: why isn't the `-c <config>` parameter convenient enough?

Reply:
when you start working with our creds helpers for docker and git. you can just set an env variable in the shell you are in and then have things work